testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js

```diff
@@ -1125,6 +1125,39 @@ QUnit.module('popup', moduleConfig, () => {
         parentContainer.remove();
     });
 
+    QUnit.test('popup is rendered at the top of element if container limited place below (T948335)', function(assert) {
+        const items = [];
+        for(let i = 0; i < 20; i++) {
+            items.push(`item ${i}`);
+        }
+
+        const windowHeight = $(window).outerHeight();
+        const parentContainer = $('<div>')
+            .attr('id', 'specific-container')
+            .css('overflow', 'hidden')
+            .height(windowHeight / 3)
+            .insertBefore('#qunit-fixture');
```

Comment:
Is it necessary to place the container exactly outside of `#qunit-fixture`? If we place the container inside of it we'll avoid container removing in the test end

Reply:
we heed use visible part of the window

Reply:
Will adding class [qunit-fixture-visible](https://github.com/DevExpress/DevExtreme/blob/21_1/testing/runner/Views/Main/RunSuite.cshtml#L60-L65) to element `#qunit-fixture` help in this case?